aspnet/visual-studio/overview/2013/creating-web-projects-in-visual-studio.md

```diff
@@ -86,7 +86,7 @@ The check box caption might be **Host in the cloud** or **Create remote resource
 
 ![Configure Azure App Settings](creating-web-projects-in-visual-studio/_static/image9.png)
 
-For more information about how to use this process for creating Azure resources, see [Get Started with Azure and ASP.NET](/azure/app-service-web/app-service-web-get-started-dotnet) and [Creating a virtual machine for a web site with Visual Studio](https://azure.microsoft.com/documentation/articles/virtual-machines-dotnet-create-visual-studio-powershell/).
+For more information about how to use this process for creating Azure resources, see [Get Started with Azure and ASP.NET](/azure/app-service-web/app-service-web-get-started-dotnet) and [Visual Studio images on Azure](https://docs.microsoft.com/azure/virtual-machines/windows/using-visual-studio-vm).
```

Comment:
The replacement URL is for a different task -- using VS by using a VM that has VS on it.  Judging from the link text, the original URL led to an article about using VS to create a VM intended to host a web site. If we can't find an article about a task similar to the original one it might be better to just delete the link and leave only the Get Started with Azure and ASP.NET link.

Reply:
@tdykstra  I deleted the proposed link. I was unable to find a similar article to replace the broken link. Per your guidance, I am leaving it with the Get Started with Azure and ASP.NET link. Thanks.